CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+1.2.7 / 2021/07/15
+===================
+- Added new method `.test()` to the `WebhookClient` class
+
```

Comment:
New method is a new feature so it should be a [semver-minor](https://semver.org/#spec-item-7) bump to `1.3.0`. Should've been done in earlier changes as well, but better late than never.